playbooks/rest_objects/nfsServer/delete.yml

```diff
@@ -1,18 +1,19 @@
 ---
 - hosts: localhost
   vars_files:
-    - ../../group_vars/all
+    - ../../../group_vars/all
   tasks:
     - name: Get nfsServer
-      nfsServer:
+      commonGetPost:
         login:
           unityIP: "{{ unity_host }}"
           username: "{{ unity_username }}"
           password: "{{ unity_password }}"
         get:
+          resource_type: nfsServer
           fields: id
-      register: result1
-    - debug: var=result1['output']['get'][0]['id']
+      register: get_nfsServer_results
+    - debug: var=get_nfsServer_results['output']['get'][0]['id']
```

Comment:
IMHO "_results" part in the variable is excessive. I think the proper form is "get_nfsServer_id"